lib/gst/plugin/gstclappersink.c

```diff
@@ -869,6 +940,12 @@ gst_clapper_sink_class_init (GstClapperSinkClass *klass)
           DEFAULT_KEEP_LAST_FRAME,
           G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
 
+  g_object_class_install_property (gobject_class, PROP_ROTATION,
+      g_param_spec_enum ("rotation", "Video rotation",
```

Comment:
Maybe lets call this "rotate-method" with `PROP_ROTATE_METHOD` for consistency with other GStreamer elements. Example: https://gitlab.freedesktop.org/gstreamer/gstreamer/-/blob/7755cd2f7a40e92f35d50765af09279384ec9a0e/subprojects/gst-plugins-bad/sys/d3d11/gstd3d11videosink.cpp#L343